src/09_dictionaries.py

```diff
@@ -34,14 +34,19 @@
 ]
 
 # Add a new waypoint to the list
-# YOUR CODE HERE
+waypoints.append({"lat": 45, "lon": -125, "name": "a fourth place"})
+print(waypoints)
 
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # Note: It's okay to access the dictionary using bracket notation on the
 # waypoints list.
 
-# YOUR CODE HERE
+waypoints[0]["lon"] = -130
+print(waypoints)
```

Comment:
As we discussed this should loop through the array of dictionaries and find the one that has the name "a place" and then change the lon of that entry to -130.